data/xml/schema.rnc

```diff
@@ -7,7 +7,10 @@ url = element url {
 }
 fixed-case = element fixed-case { MarkupText }
 tex-math = element tex-math { text }
-MarkupText = (text | b | i | url | fixed-case | tex-math )+
+a = element a {
+  attribute href { xsd:anyURI } & text
+}
+MarkupText = (text | b | i | url | fixed-case | tex-math | a)+
```

Comment:
@mjpost Can I ask that we don't make changes to the XML schema without a corresponding update to the Python library? This can break things otherwise.

Reply:
Yes. What happened is someone used the metadata corrections API to add an `<a href=` tag to their abstract. I initially went ahead with this but then decided against it, and manually converted them to `<url>` tags instead. I think I had intended to move this, too, but forgot. I agree that a change like this shouldn't be buried in an unrelated PR. I'll remove it in #5446.

Reply:
Oh, I assumed it was the submitter. It does raise the issue that our edit dialog should disallow out-of-scope tags!